addon/components/rdfa-editor-container.hbs

```diff
@@ -0,0 +1,45 @@
+{{#if @busy}}
```

Comment:
This doesn't seem to be used? If it's not needed I would delete it since it seems to be some sort of implementation detail of the editor that was copied over?

Reply:
I wanted to use it, as it would be useful for showing when loading but I didn't find a use for it yet, so I guess I will remove it